content/summits/developer/2025-Nov/_index.md

```diff
@@ -0,0 +1,167 @@
+
+{{< image >}}
+src = 'https://upload.wikimedia.org/wikipedia/commons/c/c7/ESS_logotype_cyan_S.png'
+alt = 'ESS Logo'
+height = 120
+{{< /image >}}
+
+## During the event
+
+- [Worklog]()
+- [Pre-planning issues](https://github.com/scientific-python/summit-2025-nov/issues/)
+- [Discord channel]()
+- There is also a WhatsApp group for travel coordination
+
+## Agenda
+
+<style type="text/css">
+table {
+  margin-left: 0 !important;
+}
+</style>
+
+{{< details "**Tuesday**" >}}
+| Time | Description |
+|------|-------------|
+| | Arrive |
+| 18:30 | Informal group dinner |
+{{< /details >}}
+
+{{< details "**Wednesday**" >}}
+| Time | Description |
+|------|-------------|
+| 9:00--9:30 | Breakfast |
+| 9:00--9:30 | Welcome & coordination |
+| 9:30--13:00 | Work session |
+| 13:00--14:30 | Lunch & campus/coffee walks |
+| 14:30--17:30 | Work session |
+| 17:30--18:00 | Check-in / discussion |
+{{< /details >}}
+
+{{< details "**Thursday**" >}}
+| Time | Description |
+|------|-------------|
+| 9:00--9:30 | Breakfast |
+| 9:00--9:30 | Welcome & coordination |
+| 9:30--13:00 | Work session |
+| 13:00--14:30 | Lunch & campus/coffee walks |
+| 14:30--17:30 | Work session |
+| 17:30--18:00 | Check-in / discussion |
+{{< /details >}}
+
+{{< details "**Friday**" >}}
+| Time | Description |
+|------|-------------|
+| 9:00--9:30 | Breakfast |
+| 9:00--9:30 | Welcome & coordination |
+| 9:30--13:00 | Work session |
+| 13:00--14:30 | Lunch & campus/coffee walks |
+| 14:30--17:30 | Work session |
+| 17:30--18:00 | Check-in / discussion |
+{{< /details >}}
+
+{{< details "**Saturday**" >}}
+| Time | Description |
+|------|-------------|
+| | Depart |
+{{< /details >}}
+
+</div>
+
+The event is held at [Pioneer Centre for AI - KU](https://maps.app.goo.gl/LzAeD6L2Hw1BV2gUA)
+
+## Funding
+
+Unfortunately, this year we do not have any dedicated funding to cover participants travel and accommodation costs.
+
+Project maintainers, please reach out to your individual projects or employers to determine whether they have funds to sponsor your attendance.
+
+## Logistics
+
+We have space for 20-25 participants for three days.
+This will be an invite-only event that requires upfront agreement to:
+(a) take part in at least one pre-summit planning meeting,
+(b) collaborate with fellow participants on a work plan,
+(c) attend the summit in-person, and
+(d) participate, to whatever degree possible, in several months of post-summit implementation.
+
+### Airport to hotel
+
+...
+
+### Food
+
+...
+
+### Participants
+
+Participants are recruited from the community of developers of packages
+such as NumPy, SciPy, matplotlib, xarray, pandas, scikit-image, scikit-learn,
+NetworkX, and IPython, as well as domain stacks including Astropy, Pangeo, and
+scikit-HEP.
+
+{{< details "**List of confirmed participants**" >}}
+
+- ...
+
+{{< /details >}}
+
+## About
+
+### Work Topics
+
+The 2025 November Scientific Python Developer Summit will focus on the following work topics:
```

Comment:
I'm not sure about the topics, but let's just do a workaround to start with, and then merge this PR.
```suggestion
The 2025 November Scientific Python Developer Summit will focus on work topics our participants express the most interest in. For comparison, these were the focus areas for the summit held in May, in Seattle:
```